src/app/demo/demo.module.ts

```diff
@@ -16,28 +16,28 @@ export const routes = [
     path: 'typescript',
     loadChildren: './typescript/typescript.module#TypescriptModule',
     name: 'Typescript',
-    description: 'Learn TypeScript: A language that extends JavaScript, which angular is built with.',
+    description: 'Learn TypeScript: A superset of JavaScript, which Angular is written in',
```

Comment:
I'm kind of not a big fan of "superset" here, it's very natural for everybody familiar with TS, but for those who haven't heard it used before and don't have math background can take extra mental effort.